src/components/FileDirectory.vue

```diff
@@ -44,13 +63,13 @@ export default {
   },
   computed: {
     files() {
-      return this.directory.files.map(file => {
+      return this.directory.files.map((file) => {
```

Comment:
This looks like we're still using the old directory-based upload than file based. Maybe we need to get `master` merged in?